lib/ae_mdw/db/hardfork_presets.ex

```diff
@@ -0,0 +1,54 @@
+defmodule AeMdw.Db.HardforkPresets do
+  @moduledoc """
+  Saves hardfork presets into Mdw database.
+  """
+  alias AeMdw.Database
+  alias AeMdw.Db.IntTransfersMutation
+  alias AeMdw.Db.Model
+  alias AeMdw.Db.State
+
+  @doc """
+  Imports hardfork migrated accounts.
+  """
+  @spec import_account_presets() :: :ok
+  def import_account_presets do
+    case Database.next_key(Model.KindIntTransferTx, {"accounts_lima", {-1, -1}, <<>>, -1}) do
```

Comment:
Can we turn this entirely into a mutation? That way we continue encapsulating Database calls through the State

Reply:
The `Database` is used because the goal is to check if the data is already persisted.

Reply:
I understand that's **what** it does. I was suggesting changing **how** it does it. It's optional anyway, feel free to merge it

Reply:
I am talking about `how` not to encapsulate things to make the purpose clearer.